scripts/codeship_aws_eb_deploy

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+set -e
+set -o pipefail
+
+APPLICATION_FOLDER=${1:?'You need to provide the directory with your code as the second parameter'}
+APPLICATION_NAME=${2:?'You need to provide the Elastic Beanstalk application name'}
+ENVIRONMENT_NAME=${3:?'You need to provide the Elastic Beanstalk environment name'}
+S3_BUCKET=${4:?'You need to provide the S3 Bucket to upload the artefact to'}
+
+echo "Starting ElasticBeanstalk deployment to $APPLICATION_NAME:$ENVIRONMENT_NAME"
+
+echo "CHANGING Directory to $APPLICATION_FOLDER"
+cd $APPLICATION_FOLDER
+
+ARTEFACT_NAME=${CI_COMMIT_ID}${CI_BUILD_ID}
```

Comment:
ARTEFACT_NAME is a misleading name, especially for the later uses as the version